packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -170,9 +170,9 @@ class _TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
 
   @override
   void deactivate() {
+    super.deactivate();
     if (_entry != null)
       _controller.reverse();
-    super.deactivate();
```

Comment:
can you elaborate on this change? It's unusual to do work after a deactivate, since by then you're not active and shouldn't do work...